.gitignore

```diff
@@ -5,6 +5,7 @@
 bin/
 build/
 ci/
+jacoco.exec
```

Comment:
Can you also remove the `code/edgebridge/jacoco.exec` file from source control as well?